addons/mail/static/src/core/common/attachment_list.js

```diff
@@ -102,4 +102,9 @@ export class AttachmentList extends Component {
     get isInChatWindowAndIsAlignedLeft() {
         return this.env.inChatWindow && !this.env.alignedRight;
     }
+
+    get showDeleteOnAttachmentCard() {
```

Comment:
Nitpicking: I would just name `showDelete`. More readable, and the part `OnAttachmentCard` as not much value, so better name it briefly